src/python/doc/wasserstein_distance_user.rst

```diff
@@ -72,33 +71,33 @@ any matching has a cost +inf and thus can be considered to be optimal. In such a
 
 .. testcode::
 
-    import gudhi.wasserstein
+    from gudhi.wasserstein import wasserstein_distance # Could also be: from gudhi.hera import wasserstein_distance
     import numpy as np
 
     dgm1 = np.array([[2.7, 3.7],[9.6, 14.],[34.2, 34.974], [3, np.inf]])
     dgm2 = np.array([[2.8, 4.45], [5, 6], [9.5, 14.1], [4, np.inf]])
-    cost, matchings = gudhi.wasserstein.wasserstein_distance(dgm1, dgm2, matching=True, order=1, internal_p=2)
+    cost, matchings = wasserstein_distance(dgm1, dgm2, matching=True, order=1, internal_p=2)
 
     message_cost = "Wasserstein distance value = %.2f" %cost
-    print(message_cost)
+    print(f"Wasserstein distance value = {cost:.2f}")
```

Comment:
Is message_cost still useful?

Reply:
I missed this one... Done on [274c3b8](https://github.com/GUDHI/gudhi-devel/pull/853/commits/274c3b8cef2e3dbd61ca3e9af468f7c7f07362f9)